shared/src/test/diff/nu/Annotations.mls

```diff
@@ -39,7 +48,7 @@ fun foo(n) =
   else ()
 //│ fun foo: Int -> ()
 
-// TODO
+:NoJS // recurses infinitely since `div` is float division. do not run it for now
```

Comment:
Why not just increase the threshold to `n > 0.5` instead of writing a full comment and disabling JS? :P